content/en/apps/guides/updates/incremental-updates.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "Rolling out CHT Core updates incrementally"
+linkTitle: "Incremental updates"
```

Comment:
"incremental updates" has been used to mean delivering a new cht-core version to a subset of users so it may be confusing. Can we rebrand this to "Feature Flags" which is a more industry standard name for this concept? If you agree, this will need updates throughout the doc...

Reply:
I thought there was a conversation at some point arguing against using "feature flag".... but I agree and have updated it!